aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -175,7 +175,7 @@ void CUDAGraph::instantiate() {
     // who prefer not to report error message through these arguments moving forward
     // (they prefer return value, or errors on api calls internal to the capture)
 #if (defined(CUDA_VERSION) && CUDA_VERSION >= 12000)
-    AT_CUDA_CHECK(cudaGraphInstantiate(&graph_exec_, graph_, 0));
+    AT_CUDA_CHECK(cudaGraphInstantiate(&graph_exec_, graph_, cudaGraphInstantiateFlagUseNodePriority));
```

Comment:
should we delete whole `version < 11040` branch or is it needed for ROCM? If it's the latter it should be ifdef'd on rocm, not pretend we support driver versions we don't